### PR TITLE
Add the Preprocessing folder in ElasticDL and README.md

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

File: preprocessing/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ElasticDL Preprocessing
+
+This is a feature preprocessing library provided by ElasticDL.  
+It provides API in the following forms:
```
Comment (skydoorkai) [inline preprocessing/README.md]: API -> APIs

File: preprocessing/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Keras layer
+- Feature column Api
+
+This is an extension of the native [Keras Preprocessing Layers](https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/keras/preprocessing) and [Feature Column API](https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/feature_column) from TensorFlow. We can develop our model using the native API and our extension and train this model using both Native TensorFlow and ElasticDL.
```
Comment (skydoorkai) [inline preprocessing/README.md]: What is this `native API`?

Comment (brightcoder01) [reply preprocessing/README.md]: Updated to native high level API.
It contains Keras Layers and Feature Columns

File: preprocessing/README.md (hunk 1 above)
Comment (brightcoder01) [reply preprocessing/README.md]: Good catch, updated.

**Review by workingloong — APPROVED** [review]

File: preprocessing/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# ElasticDL Preprocessing
+
+This is a feature preprocessing library provided by ElasticDL.  
+It provides APIs in the following forms:
+
+- Keras layer
+- Feature column Api
```
Comment (terrytangyuan) [inline preprocessing/README.md]: API

File: preprocessing/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (terrytangyuan) [inline preprocessing/README.md]: high-level

File: preprocessing/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+This is an extension of the native [Keras Preprocessing Layers](https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/keras/preprocessing) and [Feature Column API](https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/feature_column) from TensorFlow. We can develop our model using the native high level API from TensorFlow and our library. We can train this model using Native TensorFlow or ElasticDL.
```
Comment (terrytangyuan) [inline preprocessing/README.md]: Native -> native

File: preprocessing/README.md (hunk 3 above)
Comment (brightcoder01) [reply preprocessing/README.md]: Good catch, updated.

File: preprocessing/README.md (hunk 5 above)
Comment (brightcoder01) [reply preprocessing/README.md]: Good catch, updated.

File: preprocessing/README.md (hunk 4 above)
Comment (brightcoder01) [reply preprocessing/README.md]: Good catch, updated.

**Pull request merged**